### DISCUSSION TITLE
Is it possible to use kubectl built-in kustomize to apply transformers with helmfile?

### DISCUSSION THREAD

**Discussion by reegnz** [helmfile/helmfile#1458] [Questions]
AFAICT you can tell chartify through helmfile config to use a custom kustomize binary. Can you also instruct it somehow to use `kubectl kustomize`?

**Comment by reegnz** [comment]
I had a github issue open with chartify for a while but I thought to ask here as well, maybe someone is using helmfile this way: https://github.com/helmfile/chartify/issues/63

**Comment by yxxhero** [comment]
PR is welcome.

**Comment by reegnz** [comment]
I realized it cannot be done in a fully compatible way because `kubectl kustomize` only supports `build` but doesn't expose `edit` commands which chartify relies on.
